lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -240,24 +240,17 @@ export class UserAgentApplication {
 
     // if extraScopesToConsent is passed, append them to the login request
     let scopes: Array<string>;
-
-    if (request) {
-      if (request.extraScopesToConsent) {
-        scopes = [...request.scopes, ...request.extraScopesToConsent];
-      }
-      else {
-        scopes = request.scopes;
-      }
-    }
+    scopes = this.appendScopes(request);
```

Comment:
why not just say `let scopes = this.appendScopes(request)`

Reply:
Yeah. Can be. Must have been lost in the merge. Can edit.